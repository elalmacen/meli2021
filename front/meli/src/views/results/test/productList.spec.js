import React from 'react';
import {shallow} from 'enzyme';
import View from '../productList';

describe('test searchBox component', () => {
    test('renders without crashing', () => {
        const products = [
            {
                id: 12,
                picture: "https://http2.mlstatic.com/D_NQ_NP_848472-MLA43108239303_082020-O.webp",
                free_shipping : 'price free',
                price : {
                    amount: 20,
                    decimals: 1
                },
                title: "batman",
                region: "Villa del parque"
    
            }
        ];

        const categories = ['batman','robin'];

        const wrapper = shallow(<View products={products} categories={categories} />);
        expect(wrapper).toMatchSnapshot();
    });    
});
