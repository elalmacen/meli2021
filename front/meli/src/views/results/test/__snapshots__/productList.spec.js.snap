// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test searchBox component renders without crashing 1`] = `
<Fragment>
  <BreadCrumb
    categories={
      Array [
        "batman",
        "robin",
      ]
    }
  />
  <Container
    className="search-list"
    fluid={false}
  >
    <ol>
      <Card
        key="0"
        product={
          Object {
            "free_shipping": "price free",
            "id": 12,
            "picture": "https://http2.mlstatic.com/D_NQ_NP_848472-MLA43108239303_082020-O.webp",
            "price": Object {
              "amount": 20,
              "decimals": 1,
            },
            "region": "Villa del parque",
            "title": "batman",
          }
        }
      />
    </ol>
  </Container>
</Fragment>
`;
